src/mmda/parsers/grobid_full_text_parser.py

```diff
@@ -0,0 +1,112 @@
+"""
+
+@geli-gel
+
+"""
+from grobid_client.grobid_client import GrobidClient
+from typing import Optional, List
+
+import io
+import os
+import xml.etree.ElementTree as et
+
+from mmda.parsers.parser import Parser
+from mmda.types.annotation import BoxGroup, Box
+from mmda.types.document import Document
+from mmda.types.names import PagesField, RowsField, TokensField
+
+REQUIRED_DOCUMENT_FIELDS = [PagesField, RowsField, TokensField]
+NS = {"tei": "http://www.tei-c.org/ns/1.0"}
+
+
+class GrobidFullTextParser(Parser):
```

Comment:
can we rename this to make it clear in the name that this is an unusual Parser that takes as input also a separate `Document`? like `GrobidAugmentExistingDocumentParser`?  then it wouldnt clash if we add a `GrobidFullTextParser` later which strictly creates new documents from the Xml itself without any mapping

Reply:
would it make sense to make a new directory under mmda called `augmenters`? and name it `GrobidAugmenter`? just cause i like shorter names